ros2/source/images.yaml.em

```diff
@@ -11,48 +11,13 @@ images:
             - docker_templates
         upstream_packages:
             - bash-completion
-            - build-essential
-            - clang-format
-            - cmake
-            - cppcheck
             - git
             - libasio-dev
-            - libassimp-dev
-            - libcurl4-openssl-dev
-            - libeigen3-dev
-            - libfreetype6-dev
-            - libgles2-mesa-dev
-            - libglu1-mesa
-            - libopencv-dev
-            - libpoco-dev
-            - libqt5core5a
-            - libqt5gui5
-            - libqt5opengl5
-            - libqt5widgets5
-            - libssl-dev
-            - libtinyxml-dev
             - libtinyxml2-dev
-            - libxaw7-dev
-            - libxrandr-dev
-            - openssl
-            - pydocstyle
-            - pyflakes
-            - python3-coverage
-            - python3-dev
-            - python3-empy
-            - python3-mock
-            - python3-nose
-            - python3-pep8
-            - python3-pyparsing
-            - python3-pytest
-            - python3-pytest-cov
-            - python3-pytest-runner
-            - python3-setuptools
-            - python3-yaml
-            - qtbase5-dev
             - wget
+        downstream_packages:
+            - python-rosdep
```

Comment:
(I know it doesn't come from your changes but from my initial patch): As ROS 2 targets Python 3, can we install python**3**-rosdep here for consistency?

Reply:
Fixed, also realized downstream template was redundant